raven/src/main/java/org/opencadc/raven/RavenInitAction.java

```diff
@@ -286,6 +295,51 @@ static MultiValuedProperties getConfig() {
         }
     }
 
+    static Map<URI, StorageSiteRule> getStorageSiteRules(MultiValuedProperties props) {
+        StringBuilder sb = new StringBuilder();
+        Map<URI, StorageSiteRule> prefs = new HashMap<>();
+
+        List<String> putPreferences = props.getProperty("org.opencadc.raven.putPreference");
+        for (String putPreference : putPreferences) {
+
+            URI resourceID = null;
+            List<String> resourceIDs = props.getProperty(putPreference + ".resourceID");
+            if (resourceIDs.isEmpty()) {
+                sb.append(String.format("%s.resourceID: MISSING\n", putPreference));
+            } else if (resourceIDs.size() > 1) {
+                sb.append(String.format("%s.resourceID: MULTIPLE ENTRIES\n", putPreference));
+            } else if (!StringUtil.hasText(resourceIDs.get(0))) {
+                sb.append(String.format("%s.resourceID: EMPTY VALUE\n", putPreference));
+            } else {
+                try {
+                    resourceID = new URI(resourceIDs.get(0));
+                } catch (URISyntaxException e) {
+                    sb.append(String.format("%s.resourceID: INVALID URI\n", putPreference));
+                }
+            }
+
+            List<String> namespaces = props.getProperty(putPreference + ".namespace");
+            if (namespaces.isEmpty()) {
+                sb.append(String.format("%s.namespace: MISSING\n", putPreference));
+            } else {
+                for (String namespace : namespaces) {
+                    if (!StringUtil.hasText(namespace)) {
+                        sb.append(String.format("%s.namespace: EMPTY VALUE\n", putPreference));
+                    } else if (namespace.matches(".*[\\s].*")) {
+                        sb.append(String.format("%s.namespace: MULTIPLE ENTRIES\n", putPreference));
```

Comment:
Updated the error messages. An uppercase message previously was used since the message was a word or two long. That doesn't work well for a longer more informative message so I make them lower case. Not sure what value upper case brings to the logging. Easier to pick out maybe, but this is at startup when there isn't much logging to look through. 
We do check the syntax of the properties files, but we don't check the syntax of the property values. I added a Namespace class and a NamespaceSyntaxException class to raven for you to look at, permanent home TBD.
Baldur does not check the namespace syntax, other than making sure it's a valid regex expression. It does check that the groups are a valid GroupURI. I'll add an issue to baldur to check the namespace syntax.